packages/fiori/src/Wizard.js

```diff
@@ -302,7 +302,7 @@ class Wizard extends UI5Element {
 	}
 
 	static get CONTENT_TOP_OFFSET() {
-		return 32;
+		return 0;
```

Comment:
Scrolling from the current step to the next one, the  portion where you are between two steps and both are not marked as current is bigger then before.

Reply:
should be fixed now ... the problem is that the wizard tries to select a disabled step.

Reply:
I assume these 32 px are now covered by the change in the padding below
padding: 0.875rem **1rem;** -> padding: 0.875rem **2rem**;

Not sure why we need it, because I am not that into the issue and the Wizard design, 
but you probably have a good reason and I am fine this way as well. Probably some brief ()sanity IE testing would be good.